Vitals/ContentView.swift

```diff
@@ -77,6 +77,13 @@ struct ContentView: View {
     private static let processListHeight = 460
     private static let topPadding = 6
     private static let bottomPadding = 4
+    private static var leftPadding: Int {
+        if #available(macOS 10.0, *) {
```

Comment:
This should actually be `macOS 11.0, *` to check for Big Sur – I just decremented it to check the conditional was working as expected and forgot to update. Fixed in https://github.com/hmarr/vitals/commit/db4068fd275468d9d8ec8018234a2c301318f1ad.